docs/tutorial/window-customization.md

```diff
@@ -134,9 +150,66 @@ const win = new BrowserWindow({
 > color and dimension values from a renderer using a set of readonly
 > [JavaScript APIs][overlay-javascript-apis] and [CSS Environment Variables][overlay-css-env-vars].
 
-## Create transparent windows
+## Custom draggable regions
+
+By default, windows are dragged using the titlebar provided by the OS chrome. Apps that remove the default titlebar need to use the `app-region` CSS property to define specific areas that can be used to drag the window. Setting `app-region: drag` marks an area as draggable. Setting `app-region: no-drag` excludes an area from dragging. Note that only rectangular shapes are currently supported.
+
+To make the whole window draggable, you can add `app-region: drag` as
+`body`'s style:
+
+```css title='styles.css'
+body {
+  app-region: drag;
+}
+```
+
+And note that if you have made the whole window draggable, you must also mark
+buttons as non-draggable, otherwise it would be impossible for users to click on
+them:
+
+```css title='styles.css'
+button {
+  app-region: no-drag;
+}
+```
+
+If you're only setting a custom titlebar as draggable, you also need to make all
+buttons in titlebar non-draggable.
+
+### Tip: disable text selection
+
+When creating a draggable region, the dragging behavior may conflict with text selection.
+For example, when you drag the titlebar, you may accidentally select its text contents.
+To prevent this, you need to disable text selection within a draggable area like this:
 
-By setting the `transparent` option to `true`, you can make a fully transparent window.
+```css
+.titlebar {
+  user-select: none;
+  app-region: drag;
+}
+```
+
+### Tip: disable context menus
+
+On some platforms, the draggable area will be treated as a non-client frame, so
+when you right click on it, a system menu will pop up. To make the context menu
+behave correctly on all platforms, you should never use a custom context menu on
+draggable areas.
+
+## Frameless windows
+
+A frameless window removes all [chrome](https://developer.mozilla.org/en-US/docs/Glossary/Chrome) applied by the OS, including window controls.
```

Comment:
At this point in the document, we have already defined `chrome` above and shouldn't need to re-define it